### PR TITLE
Update 2019-10-22-react-release-channels.md

### PR DESCRIPTION
**Pull request by aroundarmor**

### CODE REVIEW

**Review by alejandronanez — CHANGES_REQUESTED** [review]
@aroundarmor this PR still needs a lot of work as there are many many inconsistencies in the translation, Please proof read the translation before calling it ready for review.

Thank you!

**Comment by alejandronanez** [comment]
@aroundarmor I'm labeling this as `invalid` in the meantime as this translation doesn't meet the minimum quality standards we have for the translations. I happily remove this label if you decide to update this Pull Request.

Thanks for your understanding.

**Comment by aroundarmor** [comment]
Hello @alejandronanez,
I would like to know more about the changes that I have to make. If you will be a little bit specific I can give it a try to resolve all the things.

File: content/blog/2019-10-22-react-release-channels.md [hunk 1]
```diff
@@ -1,62 +1,62 @@
...
 
-You can expect updates to Latest to be extremely stable. Versions follow the semantic versioning scheme. Learn more about our commitment to stability and incremental migration in our [versioning policy](/docs/faq-versioning.html).
+Puede esperar que las actualizaciones de Latest sean extremadamente estables. Las versiones siguen la semántica esquema de versionado. Obtenga más información sobre nuestro compromiso con la estabilidad y la migración incremental en nuestro [versioning policy](/docs/faq-versioning.html).
 
-### Next Channel {#next-channel}
+### Siguiente-canal {#siguiente-canal}
 
-The Next channel is a prerelease channel that tracks the master branch of the React repository. We use prereleases in the Next channel as release candidates for the Latest channel. You can think of Next as a superset of Latest that is updated more frequently.
+El Siguiente canal es un canal de presentación que pistas la rama maestra del repositorio de React. Usamos pre lanzamientos en el canal Siguiente como candidatos de lanzamiento para el Latest canal. Puede pensar en Next como un superconjunto de Latest que se actualiza con más frecuencia.
```
Comment (alejandronanez) [inline content/blog/2019-10-22-react-release-channels.md:42]: @aroundarmor this is one of the parts that need a better translation. If you proof read the whole document you'll find many occurrences like this one.

File: content/blog/2019-10-22-react-release-channels.md [hunk 2]
```diff
@@ -1,62 +1,62 @@
...
 
-Each of React's release channels is designed for a distinct use case:
+Cada uno de los canales de lanzamiento de React está diseñado para un caso de uso distinto:
 
-- [**Latest**](#latest-channel) is for stable, semver React releases. It's what you get when you install React from npm. This is the channel you're already using today. **Use this for all user-facing React applications.**
-- [**Next**](#next-channel) tracks the master branch of the React source code repository. Think of these as release candidates for the next minor semver release. Use this for integration testing between React and third party projects.
-- [**Experimental**](#experimental-channel) includes experimental APIs and features that aren't available in the stable releases. These also track the master branch, but with additional feature flags turned on. Use this to try out upcoming features before they are released.
+- [**Último**](#último-canal) Es para el estable, lanzamientos de semver React. Es lo que obtienes cuando instalas React desde npm. Este es el canal que ya estás usando hoy. **Use esto para todas las aplicaciones React orientadas al usuario.**
```
Comment (alejandronanez) [inline content/blog/2019-10-22-react-release-channels.md:22]: Please do not translate the #anchors, just the link text. 👍

**Pull request closed without merging**